packages/react/src/components/Select/Select.tsx

```diff
@@ -88,6 +88,11 @@ const Select = (props: SelectProps) => {
   const resetKeyword = () => keyword && setKeyword('');
 
   const [sortedOptions, setSortedOptions] = useState(options);
+  // Enable dynamic change of options by resetting sortedOptions
+  useUpdate(() => {
+    setSortedOptions(options);
+    setSelectedValues(multiple ? value ?? [] : []);
+  }, [options]);
```

Comment:
Denne vil bare bli trigget når referansen til `options` endrer seg. Vi må nok gjøre en dypere sammenlikning for å se om innholdet har endret seg. Vi har nylig gjort en tilsvarende endring i `CheckboxGroup`, ta en titt her: https://github.com/digdir/designsystem/blob/main/packages/react/src/components/CheckboxGroup/CheckboxGroup.tsx#L69-L76

Reply:
God observasjon som jeg ikke var klar over! Takk! 😊

Reply:
Godt poeng av @TomasEng ! 

Problemet her ligger egentlig i javascript og hvordan den behandler objekter og lister av objekter. Det samme problemet vil og gjelder for bla. React props! 

Her er noen tips mtp. fallgruver ved `useEffect/Memo/Callback` i React: https://dev.to/vicnovais/understanding-referential-equality-in-reacts-useeffect-2m7o

Reply:
Veldig nyttig artikkel - takk! 😄